src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -15,44 +16,28 @@ import { ArtworkDetails_artwork } from "__generated__/ArtworkDetails_artwork.gra
 
 export interface ArtworkDetailsProps {
   artwork: ArtworkDetails_artwork
+  user: User
+  mediator?: Mediator
 }
 
 const ArtworkDetailsContainer = Box
 const SpinnerContainer = Box
 
 export const ArtworkDetails: React.SFC<ArtworkDetailsProps> = props => {
-  const { artwork } = props
-  const renderAbout =
-    artwork.additional_information ||
-    artwork.certificateOfAuthenticity ||
-    artwork.conditionDescription ||
-    artwork.description ||
-    artwork.framed ||
-    artwork.image_rights ||
-    artwork.manufacturer ||
-    artwork.provenance ||
-    artwork.publisher ||
-    artwork.series ||
-    artwork.signatureInfo
-  if (
-    !renderAbout &&
-    !artwork.articles &&
-    !artwork.exhibition_history &&
-    !artwork.literature
-  ) {
-    return null
-  }
+  const { artwork, user, mediator } = props
```

Comment:
Yup, would grab mediator from ContextConsumer rather than passing down, even if the distance is only parent to child. The less props passing the more decoupled things can be.